test/unit/test_sign.py

```diff
@@ -75,10 +75,12 @@ def test_sct_verify_keyring_lookup_error(signer, monkeypatch):
 
     with pytest.raises(
         InvalidSCTError,
-        match="Invalid key ID in SCT: not found in current keyring",
```

Comment:
Aren't we losing a bit of our test by removing this?

Reply:
> Aren't we losing a bit of our test by removing this?

That's why I added the instance check that follows it -- we're checking that we don't change the top-level exception type with this `pytest.raises`, and then the following check makes sure that the specific exception type is the one reflecting a missing key ID.

(The `match` string no longer exists -- it's baked into the diagnostics now.)